src/store/user-store.ts

```diff
@@ -16,16 +19,46 @@ class UserStore {
     return !!this.user?.id;
   }
 
+  public init = async () => {
+    this.isInitLoading = true;
+    this.error = '';
+
+    try {
+      const userID = localStorage.getItem(LSKeys.USER_ID);
+      if (userID) {
+        const response = await customerService.getCustomerByID(userID);
+        runInAction(() => {
+          console.log(response);
+          this.user = response;
+        });
+      }
+    } catch (error) {
+      runInAction(() => {
+        this.error = error instanceof Error ? error.message : String(error);
+      });
+    } finally {
+      runInAction(() => {
+        this.isInitLoading = false;
+      });
+    }
+  };
+
   public login = async (
-    creds: Pick<Customer.Profile, 'email' | 'password'>
+    customer: Pick<Customer.Profile, 'email' | 'password'>
   ) => {
     this.isPending = true;
     this.error = '';
 
     try {
-      const response = await customerService.loginCustomer(creds);
+      const tokenResponse = await authService.getUserToken(customer);
+      saveTokenToLS(tokenResponse, LSKeys.USER_TOKEN);
+      const response = await customerService.loginCustomer(customer);
       runInAction(() => {
         this.user = response.customer;
+        localStorage.setItem(
+          LSKeys.USER_ID,
+          JSON.stringify(response.customer.id)
+        );
```

Comment:
вот тут как раз есть смысл оставить response. а с токеном такая же ситуация
```suggestion
      const token = await authService.getUserToken(customer);
      saveTokenToLS(token, LSKeys.USER_TOKEN);

      const response = await customerService.loginCustomer(customer);

      runInAction(() => {
        this.user = response.customer;
        
        localStorage.setItem(
          LSKeys.USER_ID,
          JSON.stringify(response.customer.id)
        );
```